src/main/java/edu/byu/cs/service/UserService.java

```diff
@@ -126,15 +109,4 @@ private static RepoUpdate verifyRepoIsAvailableForUser(String url, String netId)
         }
         return null;
     }
-
-    /**
-     * Cleans up and returns the provided GitHub Repo URL for consistent formatting.
-     */
-    private static String requireCleanRepoUrl(String url) throws BadRequestException {
```

Comment:
This function isn't "unnecessary" just because it's currently only used once. It encapsulates and adapts the function call to another class. Previously, it was called multiple times. Now that it's currently only called once, that doesn't make it redundant. It just makes it easier to extend and reintroduce in the future.

Reply:
The function was static and only created an instance to call an instance method only ever called within this method, making this function a wrapper method for what should just be a static method